content/en/docs/getting-started/_index.md

```diff
@@ -21,23 +21,55 @@ To run MicroShift, you need a machine with at least:
 
 For production deployments, we recommend (and only test) deploying MicroShift on RHEL 8, CentOS Stream 8, or Fedora 34+ using one of two methods:
 
-- running containerized on Podman
+- running containerized with `Podman`
 - installing via .rpm (e.g. for embedding MicroShift into an `rpm-ostree` image)
 
-Both methods feature a minimal resource footprint, a strong security posture, the ability to restart/update without disrupting workloads, and optionally auto-updates.
```

Comment:
Why remove this? Intention of this is/was to clarify what differentiates "production" from the AIO case and why users shouldn't use AIO for anything but ephemeral environments.

Reply:
Ah, sorry, reading further I see you've moved it down. I think here is the right place, though, as it's more as intro and to contrast with AIO rather than explaining the diff between rpm and Podman installation.

Reply:
moved it back!